library/src/scripts/flyouts/DropDownSwitchButton.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+import { visibility } from "@library/styles/styleHelpersVisibility";
+import classNames from "classnames";
+import ButtonLoader from "@library/loaders/ButtonLoader";
+import { CheckCompactIcon } from "@library/icons/common";
+import { dropdownSwitchButtonClasses } from "@library/flyouts/dropDownSwitchButtonStyles";
+import DropDownItemButton from "@library/flyouts/items/DropDownItemButton";
+
+export interface IButtonSwitch {
+    status: boolean;
+    isLoading?: boolean;
+    onClick?: (event: any) => void;
+    label: string;
+}
+/**
+ *
+ */
+export default function DropDownSwitchButton(props: IButtonSwitch) {
+    const visibilityClasses = visibility();
+    const classes = dropdownSwitchButtonClasses();
+
+    const { isLoading, status, onClick, label } = props;
+
+    const content = (
+        <>
+            <span className={classes.itemLabel}>{label}</span>
+            <span className={classes.checkContainer}>
+                {isLoading ? (
+                    <ButtonLoader />
+                ) : (
+                    <CheckCompactIcon
+                        aria-hidden={true}
+                        className={classNames(
+                            { [visibilityClasses.visuallyHidden]: !status },
```

Comment:
Where's the "on"/"off" accessible text we discussed? Also, you're using the visuallyHidden class the wrong way. It's for making things visible to screen readers. You're making the icon hidden to sighted users, but visible to blind users. If it's not there, just don't render it. `!status && <CheckCompactIcon...`